yjit/src/backend/arm64/mod.rs

```diff
@@ -562,67 +562,45 @@ impl Assembler
 
         /// Emit a conditional jump instruction to a specific target. This is
         /// called when lowering any of the conditional jump instructions.
-        fn emit_conditional_jump<const CONDITION: u8>(cb: &mut CodeBlock, target: Target) {
+        fn emit_conditional_jump<const CONDITION: u8, const INVERSE: u8>(cb: &mut CodeBlock, target: Target) {
```

Comment:
IMO you should implement a `negate` method for your condition enum. Then you wouldn't have to specify both the condition and its inverse. Seems more succint and less error-prone.

Reply:
I pushed a branch up for this. I called the function `inverse` rather than `negate` because it felt better to me. But happy to change it.

https://github.com/Shopify/ruby/compare/better-bcond...Shopify:ruby:inverse-condition?expand=1